2023-03_sprint/sprint_parties/contributors.md

```diff
@@ -110,4 +110,16 @@ Brooklyn, New York
 [{fab}`linkedin`](https://www.linkedin.com/in/saitejaswie/)
 [{fab}`github`](https://github.com/vsaitejaswie)
 :::
+
+:::{grid-item}
+
+<img alt="Image for contributor template" src="https://media.licdn.com/dms/image/C4D03AQECXsfatPDn7Q/profile-displayphoto-shrink_800_800/0/1638788712245?e=1685577600&v=beta&t=LXFC6Qlo6ZxSOWjO5ryo7hTfnd3M2dJM5KXVeyE2KeA" class="rounded-circle">
```

Comment:
```suggestion
<img alt="" src="https://media.licdn.com/dms/image/C4D03AQECXsfatPDn7Q/profile-displayphoto-shrink_800_800/0/1638788712245?e=1685577600&v=beta&t=LXFC6Qlo6ZxSOWjO5ryo7hTfnd3M2dJM5KXVeyE2KeA" class="rounded-circle">
```

see also https://github.com/pymc-devs/pymc-data-umbrella/issues/226, trying to make things a bit easier for future us.